patroni/dcs/__init__.py

```diff
@@ -318,8 +318,14 @@ def tags(self) -> Dict[str, Any]:
 
     @property
     def nofailover(self) -> bool:
-        """The value for ``nofailover`` in :attr:`Member`.tags`` if defined, otherwise ``False``."""
-        return self.tags.get('nofailover', False)
+        """The value for ``nofailover`` in :attr:`Member`.tags`` if defined. Otherwise, returns True
```

Comment:
```suggestion
        """The value for ``nofailover`` in :attr:`Member`.tags`` if defined. Otherwise, returns ``True``
```

Reply:
First line of a docstring must be a one line summary. A more detailed explanation can be given after a line break.